CODEOWNERS

```diff
@@ -41,4 +41,8 @@
 */generators/csharp @ron-debajyoti
 
 # Language Champions for Dart and it's presets
-*/generators/dart
\ No newline at end of file
+*/generators/dart
+
+# Language Champions for Rust and its presets
+*/generators/rust @leigh-johnson
```

Comment:
@jonaslagoni Just noticed this warning message on my end. Looks like I need to be added to the repo for this check to pass?

<img width="1011"
<img width="1055" alt="Screen Shot 2022-08-02 at 9 10 55 AM" src="https://user-images.githubusercontent.com/2601819/182422408-9ea7ee39-c4f4-4df6-a19c-08a314d62e5d.png">
 alt="Screen Shot 2022-08-02 at 9 10 48 AM" src="https://user-images.githubusercontent.com/2601819/182422378-c9071ea0-68cc-4d43-83db-a04bda2b31f1.png">

Reply:
Yes, once merged I will add you as codeowner 🙂 So we can ignore this for now 😄